x-pack/plugins/ingest_manager/server/services/epm/packages/install.ts

```diff
@@ -326,3 +339,22 @@ export async function ensurePackagesCompletedInstall(
   await Promise.all(installingPromises);
   return installingPackages;
 }
+
+export function getInstallType({
+  pkgVersion,
+  installedPkg,
+}: {
+  pkgVersion: string;
+  installedPkg: SavedObject<Installation> | undefined;
+}): InstallType | undefined {
```

Comment:
Should we error or otherwise note we don't understand this condition, instead of returning undefined?

Reply:
sure I can throw an error.  the error should never exist.